io.openems.edge.battery.fenecon.home/src/io/openems/edge/battery/fenecon/home/FeneconHomeBatteryImpl.java

```diff
@@ -222,7 +243,7 @@ protected ModbusProtocol defineModbusProtocol() throws OpenemsException {
 						m(new UnsignedWordElement(507)) //
 								.m(FeneconHomeBattery.ChannelId.BATTERY_RACK_CURRENT,
 										ElementToChannelConverter.SCALE_FACTOR_MINUS_1) // [mV]
-								.m(Battery.ChannelId.CURRENT, ElementToChannelConverter.SCALE_FACTOR_MINUS_1) // [V]
+								.m(Battery.ChannelId.CURRENT, ElementToChannelConverter.SCALE_FACTOR_MINUS_1) // [mA]
```

Comment:
@huseyinsaht  `Battery.ChannelId.CURRENT` is defined as [A]; [mV] for BATTERY_RACK_CURRENT also seems wrong.

Reply:
Yes, unit should be Ampere